src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPL.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Description": "High Performance Linkpack Performance workload",
+  "Parameters": {
+    "AccountKey": "[secret:commandline]=StorageAccountKey"
```

Comment:
This is not needed anymore, if you copied it from some other profile, you could delete from there too

Reply:
Any specific reason why we are moving away from having account key parameters in profile?